doc/dftb+/manual/dftbp.tex

```diff
@@ -5403,6 +5403,18 @@ \section{hamsqrN.dat, oversqr.dat}
   \end{itemize}
 \end{itemize}
 
+Each column / row of a quadratic dense matrix belongs to an (atom, orbital)
+index pair, with the orbital index running faster. The order of the atoms
+follows their order in the input geometry. The orbitals are ordered by
+increasing ($l$, $m$) angular quantum numbers of the real tesserals with $m$
```

Comment:
Not necessarily, as the l ordering is depending on shell ordering (in the case of extended basis specification).

Reply:
```suggestion
increasing magnetic quantum number ($m$) of the angular shell ($l$), using real tesserals functions. The angular shells
are ordered according to the input to the calculation. In most cases this leads to orbitals ordered by increasing ($l$, $m$), but for non-minimal or reordered basis functions the shell order ($l$) may not necessarily be in increasing order.
```